.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-- 7.1
```

Comment:
Why drop PHP 7.1?

Reply:
Laravel 7 drops support for it as its EOL: https://www.php.net/supported-versions.php

Reply:
@tomschlick FWIW, Redhat and Debian may backport security fixes to older PHP versions long after PHP Group declares them EOL

Reply:
@paulrrogers thats true, but I've also required Laravel 7 in this release to avoid dependency hell with testbench / phpunit and since 7 now requires `>= 7.2.5` it would error on install. This update will have to be a major release.